vmaas_sync/advisory_sync.go

```diff
@@ -37,6 +37,14 @@ func syncAdvisories(syncStart time.Time, modifiedSince *string) error {
 			"advisories_sync_duration", utils.SinceStr(advSyncStart, time.Second)).
 			Debug("Downloaded advisories")
 	}
+
+	if modifiedSince != nil {
```

Comment:
For safety, please add `ENABLE_ADVISORIES_COUNT_CHECK=true` env var so we could disable this option in case of any issues.